screens/SettingsScreen.js

```diff
@@ -241,8 +247,20 @@ class SettingsScreen extends Component {
             [
                 {
                     text: ok,
-                    onPress: () =>
-                        this.props.toggleSettingsSwitch(id, this.props.token)
+                    onPress: () => {
+                        if (id === 11) {
+                            this.props.saveSettings(
+                                { id: 11, key: 'picked_up' },
+                                !this.props.user.picked_up,
```

Comment:
If this is meant to be negative, it needs a comment explaining why

Reply:
Because `saveSetting(data, value, token)` and while toggling switch the value we send to change to backend api should be opposite of current value.